### PR TITLE
Introduces the Style Line of equipment

### PR DESCRIPTION
**Pull request by lKiseki**
## About The Pull Request
Introduces the **Style** line of equipment. As its name may allude to, this line of equipment is specifically designed to prioritize style without compromising gameplay. It includes modular armors that accept common modules, plus three new modules designed specifically to alter the armor values of any given Style piece of equipment.
Currently, there are only two armors that form part of this line. More may be added later down the line if spriters feel like it.
##
### Style Line: The Rundown
- Features the Leather Jacket and the Duster, both of which have their own variants that can be acquired by using green facepaint on them.
  - The Duster is a new armor option that is part of the Style line. It has eight variants: drab, snow, black, desert, red, blue, purple, and gold.

dustergaming
### Sprites courtesy of Fitz Sholl, from our Discord.
_( Dunno if they'd like me to add their Discord handle, but yeah. Credit where it's due, since the sprites aren't mine. )_
- These have their own category in the Armor vendor, dubbed "Style Line".
- Adds Equalizer armor modules. 
  - Equalizer modules alter the stats of armor to match that of existing armor presets (light, medium, heavy). These can only be used by the Style Line.
  - By default, the Style line of equipment has no armor and no movement penalties. 
- Existing storage and armor modules are compatible with this line of equipment.
  - Equalizer modules occupy their own equipment slot, meaning other modules remain usable with an Equalizer.
- The reinforced leather jacket and reinforced leather jacket with webbing have been refactored to become a single modular armor, now part of the Style line.
  - The webbing variant can now be acquired by using green facepaint on a Leather Jacket.
  - The leather jacket can be found in the new armor vendor category.

## Why It's Good For The Game
- Gives players a greater amount of freedom in terms of style without compromising important aspects of gameplay, such as armor and movement speed.
- Adds more armor options for player loadouts, all of which are compatible with existing storage and armor modules, making them as viable as any other option.

## Changelog
:cl: Lewdcifer
add: Introducing the Style line of equipment, designed to prioritize style without compromising gameplay.
add: Includes modular armors that accept existing storage and armor modules, available under their own category in standard armor vendors.
add: Three new armor modules have been added: the Equalizer modules. These only work with the Style line, and will alter their stats to match those of existing armor presets (light, medium, heavy).
add: All options have existing variants, accessible by using green facepaint on them.
add: A new armor addition to the game, the Duster, is part of this line of equipment.
refactor: The reinforced leather jacket and its webbing variant have been turned into a single item and merged into the Style line. Its webbing variant can be accessed by using green facepaint.
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]
TGMC NOW FEATURING DANTE FROM THE DEVIL MAY CRY SERIES

**Comment by lKiseki** [comment]
image

**Comment by RaidarV** [comment]
Dante Laugh

TGMC'S NEW EXPANSION NOW FEATURING MALESTATIC "Tonī Reddogureibu" FROM HIT SERIES "UNGA MAY COPE™®©"

**Comment by Tupinambis** [comment]
We RimWorld now

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 1]
```diff
@@ -425,6 +425,38 @@
 	affected.remove_filter("eshield")
 	affected.add_filter("eshield", 1, outline_filter(1, new_color))
 
+/obj/item/armor_module/module/style
+	name = "\improper Armor Equalizer"
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/attachments/modules.dm:429]:
```suggestion
	name = "armor equalizer"
```

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 2]
```diff
@@ -425,6 +425,38 @@
...
+	<BR>The <b>Style Line</b> is a line of equipment designed to provide as much style as possible without compromising the user's protection.
+	This line of equipment accepts <b>Equalizer modules</b>, which allow the user to alter any given piece of equipment's protection according to their preferences.<BR>
+	<BR>This is an <b>Equalizer module</b>. Equalizer modules create an invisible mesh over the user's body that grants protection against many dangers, adjusting itself in such a way that their movements remain unimpeded."}
+
+/obj/item/armor_module/module/style/get_mechanics_info()
+	. = ..()
+	. += jointext(codex_info, "<br>")
+
+/obj/item/armor_module/module/style/light_armor
+	name = "\improper Light Armor Equalizer"
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/attachments/modules.dm:446]:
```suggestion
	name = "light armor equalizer"
```

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 3]
```diff
@@ -425,6 +425,38 @@
...
+	soft_armor = list(MELEE = 40, BULLET = 60, LASER = 60, ENERGY = 50, BOMB = 50, BIO = 50, FIRE = 50, ACID = 50)
+	slowdown = 0.3
+
+/obj/item/armor_module/module/style/medium_armor
+	name = "\improper Medium Armor Equalizer"
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/attachments/modules.dm:451]:
```suggestion
	name = "medium armor equalizer"
```

File: code/modules/clothing/modular_armor/modular.dm [hunk 4]
```diff
@@ -804,6 +804,89 @@
...
+	var/codex_info = {"<BR>This item is part of the <b>Style Line.</b><BR>
+	<BR>The <b>Style Line</b> is a line of equipment designed to provide as much style as possible without compromising the user's protection.
+	This line of equipment accepts <b>Equalizer modules</b>, which allow the user to alter any given piece of equipment's protection according to their preferences.<BR>"}
+
+/obj/item/clothing/suit/modular/style/get_mechanics_info()
+	. = ..()
+	. += jointext(codex_info, "<br>")
+
+/obj/item/clothing/suit/modular/style/leather_jacket
+	name = "\improper Leather Jacket"
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/modular.dm:854]:
```suggestion
	name = "leather jacket"
```

File: code/modules/clothing/modular_armor/modular.dm [hunk 5]
```diff
@@ -804,6 +804,89 @@
...
+		slot_r_hand_str = 'icons/mob/items_righthand_1.dmi',
+	)
+	icon_state_variants = list(
+		"normal",
+		"webbing",
+	)
+	current_variant = "normal"
+
+/obj/item/clothing/suit/modular/style/duster
+	name = "\improper Duster"
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/modular.dm:871]:
```suggestion
	name = "duster"
```

File: code/modules/clothing/modular_armor/modular.dm [hunk 6]
```diff
@@ -804,6 +804,89 @@
...
+	desc = "A light, loose-fitting long coat, for those that want to have more style."
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/modular.dm:872]:
```suggestion
	desc = "A light, loose-fitting colorable long coat, for those that want to have more style."
```

**Comment by wezsel11** [comment]
stops time and charges into the xeno with his japanese katana. The blade is inserted into the exoskeleton of the xeno piercing its organs. Vergilli teleports behind the xeno and returns time to normal. The xeno behind him explodes into a wave of gore. Xeno blood rains for a few seconds but all the drops seems to have missed Vergilli. Not one drop has fallen upon him and he laughs as he brushes off the dust from his shoulder. I says with a quiet voice: "You should have been more motivated."

**Review by Kuro020 — CHANGES_REQUESTED** [review]
you had to add in more colors didnt you, anyways
image
this shit is neon, desaturate it to be in line with the red and blue

**Comment by shelteredfreak** [comment]
speed mergngfefneufn plz

**Comment by AtlasRules** [comment]
I will pay whoever sprited these good money for two piece suits.

**Comment by shelteredfreak** [comment]
bonus points if I can add arm plates and leg plates to it so I can have the Elite Riot Gear from Fallout New Vegas

Now I just need to find a helmet to match with a swat mask to top it all off....

**Review by Kuro020 — APPROVED** [review]

**Comment by Tailiik** [comment]
> bonus points if I can add arm plates and leg plates to it so I can have the Elite Riot Gear from Fallout New Vegas
> 
> Now I just need to find a helmet to match with a swat mask to top it all off....

That would be cool except jaeger is going bye bye :'c 

Looks neat though

**Comment by Tupinambis** [comment]
Should either only be one armor type or have different sprites for different levels of armor.

Also I thought we were doing away with non standard colors on armor.

**Comment by Coastfront** [comment]
> > bonus points if I can add arm plates and leg plates to it so I can have the Elite Riot Gear from Fallout New Vegas
> > Now I just need to find a helmet to match with a swat mask to top it all off....
> 
> That would be cool except jaeger is going bye bye :'c
> 
> Looks neat though


Just make its tyr appearance have arm and leg plates for fallout new vegas.

**Comment by novaepee** [comment]
image

xeno winrates died because of drip

**Comment by RaidarV** [comment]
image

YOU CANNOT KILL MEEEE

I AM UNGAAA

**Comment by Aporhtonoma** [comment]
beautiful but uuuh where pink?

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**